client/guidestours/style.scss

```diff
@@ -58,3 +84,47 @@ $zoom-scale: 5; // the multiplier determining the size of the animated rings
 .guidestours__pointer__ring:after {
 	animation-delay: #{ $animation-speed / 4 };
 }
+
+.guidestours__buttonrow__choice {
+	button {
+		width: 48%;
+	}
+	button:nth-child(1) {
+		margin-right: 4%;
+	}
+	button.guidestours__button__secondary {
+		background: $gray-light;
+		color: darken( $gray, 20% );
+	}
+}
+
+.guidestours__buttonrow__single {
+	button {
+		width: 100%;
+	}
+}
+
+.guidestours__step__extra,
```

Comment:
These classes need to match our guidelines (only one double underscore separator) like `component__element`.